lib/elixir/lib/supervisor/spec.ex

```diff
@@ -75,6 +75,10 @@ defmodule Supervisor.Spec do
     * `:transient` - the child process is restarted only if it
       terminates abnormally, i.e., with an exit reason other than
       `:normal`, `:shutdown` or `{:shutdown, term}`
+  
+  Notice that supervisor that reached `max_restart_intensity` will exit with `:shutdown` reason. 
+  In this cases childs that have `:transient` restart value in their child spec will not be restarted, 
```

Comment:
@AndrewDryga I believe it should be "children" as it is plural of "child". :bowtie: